map/pom.xml

```diff
@@ -62,10 +74,48 @@
             <artifactId>jmock-legacy</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.vividsolutions</groupId>
+            <artifactId>jts</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-shapefile</artifactId>
+            <version>15.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-epsg-hsql</artifactId>
+            <version>15.2</version>
+        </dependency>
+
+
     </dependencies>
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>jaxb2-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>xjc</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <sources>
+                        <source>src/test/resources/openlr/map/mockdb/SimpleMockDatabase.xsd</source>
+                    </sources>
+                    <clearOutputDir>false</clearOutputDir>
+                    <outputDirectory>${project.build.sourceDirectory}</outputDirectory>
```

Comment:
Usually generated sources go under `${project.build.directory}/generated-sources/jaxb`.  The source directory is left unchanged by the build.  Then you don't have to add extra `.gitignore` rules to exclude them.  I can show you how to do this.

Reply:
The reason why we keep it here is because of coverall

Reply:
9921bc46eaef1f2b022379c9e24cba52db5b1735